cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -17,18 +17,24 @@ resource:
     requests:
       cpu: 2
       memory: 2Gi
+    limits:
+      cpu: 2
+      memory: 2Gi 
   tserver:
     requests:
       cpu: 2
       memory: 4Gi
+    limits:
+      cpu: 2
+      memory: 4Gi
 
 replicas:
   master: 3
   tserver: 3
 
 partition:
-  master: 3
-  tserver: 3
+  master: 0
```

Comment:
What does the "partition" field do? And is 0 fine?

Reply:
This partition doesn't have anything to do with the master replication factor. This is used specifically just for rolling upgrades. We changed it to 0 since we cannot control rolling upgrades with custom intervals in CE for K8s.